pkg/api/security/security.go

```diff
@@ -202,9 +203,56 @@ func GetClusterAgentAuthToken() (string, error) {
 	return authToken, validateAuthToken(authToken)
 }
 
+// GetCLCRunnerAuthToken load the authentication token from:
+// 1st. the configuration value of "cluster_check_runner_auth_token" in datadog.yaml
+// 2nd. from the filesystem
+// If using the token from the filesystem, the token file must be next to the datadog.yaml
+// with the filename: cluster_check_runner_auth_token
+func GetCLCRunnerAuthToken() (string, error) {
+	authToken := config.Datadog.GetString("cluster_check_runner_auth_token")
+	if authToken != "" {
+		log.Infof("Using configured cluster_check_runner_auth_token")
```

Comment:
Will this be log everytime with `ValidateCLCRunnerRequest`?

Reply:
nope, we're calling the `GetCLCRunnerAuthToken` function defined in `pkg/api/util/util.go` to validate requests, not this function.